sentry.client.config.js

```diff
@@ -0,0 +1,25 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn:
+    SENTRY_DSN ||
+    "https://6baf0213093a428d9e9ec7ae0979868c@o287463.ingest.sentry.io/5387583",
+  // Adjust this value in production, or use tracesSampler for greater control
+  tracesSampleRate: 1.0,
```

Comment:
`tracesSampler`? did you look into it?

Reply:
Yeah I did. https://docs.sentry.io/platforms/javascript/configuration/sampling/
I left this to 100% as I'm not sure to see the overload from it, and I think it's best to fine tune from 100% down to a preferred value instead of the other way around.
What do you think?

Ps: Now that I looked into it, I may have misunderstood the feature. I guess it's `x%` of **all** transactions, meaning Sentry play the roll of a tracing tool, instead of `x%` of all **failed** transactions.